tests/Resources/Common.robot

```diff
@@ -151,5 +151,17 @@ Skip If RHODS Version Greater Or Equal Than
        Skip If    condition=${version-check}==True    msg=This test is skipped for RHODS ${version} or greater
     END
 
-
-
+Get Domain From Current URL
+    [Documentation]    Gets the lowest level domain from the current URL
+    ${current_url} =    Get Location
+    ${domain} =    Fetch From Left    string=${current_url}    marker=.
+    [Return]    ${domain}
+
+Is Current Domain Equal To
+    [Documentation]    Compare the lowest level domain to a given string
+    ...   and returns True/False
+    [Arguments]    ${url}
```

Comment:
This keyword seems similar to https://robotframework.org/SeleniumLibrary/SeleniumLibrary.html#Location%20Should%20Contain

Reply:
Mh, I disagree. The Selenium keyword seems to be ok with the text being in any portion of the URL, instead this keyword explicitly checks the lowest level domain.
It might give us the same result for now, but we don't know how these URLs are going to evolve over time. I guess using the Selenium keyword with the `https` prefix anchors it to the beginning or the URL, but I'd rather use a keyword we have full control over

Reply:
ok!